test/k8s-local-cluster-test/provision-cluster

```diff
@@ -7,7 +7,12 @@ TEST_ID=$(uuidgen | cut -d'-' -f1 | tr '[:upper:]' '[:lower:]')
 CLUSTER_NAME_BASE=$(uuidgen | cut -d'-' -f1 | tr '[:upper:]' '[:lower:]')
 OVERRIDE_PATH=0
 KIND_CONFIG_FILE=$SCRIPTPATH/kind-three-node-cluster.yaml
+use_psp=false
```

Comment:
I also reviewed this yesterday, and my only question was "where are we originally setting use_psp?" Apparently it was right here all along, and I just scrolled past it. 😒